thesis/behaviour.som

```diff
@@ -0,0 +1,45 @@
+class Behaviour usingPlatform: platform = Value (
```

Comment:
I want them in my local fork and thought you could refuse them when merging with the main.